source/docs/software/labView/resources/using-the-compressor-in-labview.rst

```diff
@@ -0,0 +1,35 @@
+Using the Compressor in LabVIEW
+===============================
+
+.. image::images/ni-logo.png
+
+This snippet shows how to set up your roboRIO project to use the Pneumatic Control Module (PCM).  The PCM automatically starts and stops the compressor when specific pressures are measured in the tank. In your roboRIO program, you will need to add the following VIs.
+
+For more information, check out the following links:
+
+https://rps01.usfirst.org/frc/manual/2015/2015FRCPneumaticsManual.pdf
```

Comment:
Don't just use the raw links, perhaps do something like?

FRC Pneumatics Manual

PCM User's Guide

Pneumatics Step by Step for the roboRIO